modules/apps/headless/headless-admin-workflow/headless-admin-workflow-test/src/testIntegration/java/com/liferay/headless/admin/workflow/resource/v1_0/test/WorkflowDefinitionResourceTest.java

```diff
@@ -173,6 +173,46 @@ public void testGraphQLGetWorkflowDefinitionsPage() throws Exception {
 					workflowDefinitionsJSONObject.getString("items"))));
 	}
 
+	@Override
+	@Test
+	public void testPostWorkflowDefinition() throws Exception {
+		super.testPostWorkflowDefinition();
+
+		// content is in json format
+
+		WorkflowDefinition randomWorkflowDefinition1 =
+			randomWorkflowDefinition();
+
+		randomWorkflowDefinition1.setContent(
+			WorkflowDefinitionTestUtil.getContent(
```

Comment:
You are setting the content automatically, I am not sure if this test is testing what you want to test, the `getContent` method is always returning a `String`